docs/articles/documentation/test-api/http-authentication.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+title: HTTP Authentication
+permalink: /documentation/test-api/http-authentication.html
+checked: true
+---
+# HTTP Authentication
+
+TestCafe allows you to test web pages that are protected with HTTP [Basic](https://en.wikipedia.org/wiki/Basic_access_authentication)
+or [Windows (NTLM)](https://en.wikipedia.org/wiki/Integrated_Windows_Authentication) authentication.
+Use the `test.httpAuth` method to specify the credentials to be used by an individual test
+and the `fixture.httpAuth` method to specify the credentials for the entire fixture.
+
+```text
+fixture().httpAuth( credentials )
```

Comment:
No, we do this only when the described function can be a template string (`page`). We don't do this in `beforeEach`,  `afterEach`, etc.